data/json/professions.json

```diff
@@ -825,7 +825,7 @@
     "type": "profession",
     "ident": "maid",
     "name": { "male": "Butler", "female": "Maid" },
-    "description": "You worked in a wealthy household, but after the Cataclysm they took a family vacation to an unknown place, leaving you to fend for yourself.",
+    "description": "You were hired to take care of the housekeeping for a wealthy family.  Naturally, when things got bad, they all took off on an unplanned vacation and left you to your fate.",
```

Comment:
This is a reference to the necropolis, AFAIK. I think there'd be some value in it being preserved, but I don't feel strongly that way.